packages/commerce-sdk-react/src/auth/index.test.ts

```diff
@@ -387,20 +389,11 @@ describe('Auth', () => {
     })
 
     test.each([
-        // When user has not selected DNT pref
-        {defaultDnt: true, dw_dnt: undefined, expected: {dnt: true}},
-        {defaultDnt: false, dw_dnt: undefined, expected: {dnt: false}},
-        {defaultDnt: undefined, dw_dnt: undefined, expected: {}},
-        // When user has selected DNT, the dw_dnt cookie sets dnt
-        {defaultDnt: true, dw_dnt: '0', expected: {dnt: false}},
-        {defaultDnt: false, dw_dnt: '1', expected: {dnt: true}},
-        {defaultDnt: false, dw_dnt: '0', expected: {dnt: false}}
-    ])('dnt flag is set correctly', async ({defaultDnt, dw_dnt, expected}) => {
+        {defaultDnt: true, expected: {dnt: true}},
+        {defaultDnt: false, expected: {dnt: false}},
+        {defaultDnt: undefined, expected: {}}
+    ])('dnt flag is set correctly', async ({defaultDnt, expected}) => {
```

Comment:
Are these tests having the same description as `dnt flag is set correctly`? When one of them fails, it may be hard to know which one is failing.